### PR TITLE
Return empty string for null/undefined nodes

### PR DESCRIPTION
**Pull request by AdamBoxall**

### CODE REVIEW

File: skin-deep.js [hunk 1]
```diff
@@ -156,6 +156,10 @@ function findNodes(node, predicate, findOne) {
 }
 
 function getTextFromNode(node) {
+  // Ignore null/undefined children
+  if (node === null || node === undefined) {
```
Comment (glenjamin) [inline skin-deep.js]: Should do `if (!node && node !== 0)` to include false as well.

Comment (AdamBoxall) [inline skin-deep.js]: False is technically an object so it doesn't generate the error when properties are checked against :)

Comment (AdamBoxall) [inline skin-deep.js]: I think React also treats false special?

Comment (glenjamin) [inline skin-deep.js]: Fair enough, if you add to the test case I'll merge with that.

Comment (glenjamin) [inline skin-deep.js]: False doesn't render, so I think empty string is fine for this method.

Comment (AdamBoxall) [inline skin-deep.js]: Done

**Pull request merged**